FlowCal/mef.py

```diff
@@ -308,38 +370,44 @@ def sc_fun(p,x):
             beads_model_str,
             beads_params_names)
 
-def plot_standard_curve(fl_channel,
+def plot_standard_curve(fl_au,
                         fl_mef,
                         beads_model,
                         std_crv,
+                        xscale='linear',
+                        yscale='linear',
                         xlim=None,
                         ylim=(1.,1e8)):
     """
     Plot a standard curve with fluorescence of calibration beads.
 
     Parameters
     ----------
-    fl_channel : array_like
-        Fluorescence of the calibration beads' subpopulations, in channel
-        numbers.
+    fl_au : array_like
+        Fluorescence of the calibration beads' subpopulations, in arbitrary
+        units.
     fl_mef : array_like
         Fluorescence of the calibration beads' subpopulations, in MEF
         units.
     beads_model : function
         Fluorescence model of the calibration beads.
     std_crv : function
-        The standard curve, mapping channel units to MEF units).
+        The standard curve, mapping arbitrary units to MEF units.
 
     Other parameters:
     -----------------
+    xscale : str, optional
+        Scale of the x axis, either ``linear`` or ``log``.
+    yscale : str, optional
+        Scale of the y axis, either ``linear`` or ``log``.
```

Comment:
I feel like maybe the standard curve should have the option to be plotted on `logicle` scales to be consistent with the rest of the plots (again, attempting to unify the muscle memory / intuition we develop when analyzing flow data on plots generated by our package). haven't looked at a standard curve on those scales, though, so not sure if that is a bad idea for some unforeseen reason